Resources/misc/tag-steps.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
I'd move this to its own commit. Not sure which is best though: first or last commit ? If it is first, changing any of it will require that we rebase all of the other commits. Maybe it is best if it is last ? It doesn't even need to be in the steps tags.